### PR TITLE
WIP: Port cmdLineTester_pltest_j9sig_ext Test

### PR DESCRIPTION
**Pull request by nikolamilijevic1**
-added cmdLineTester_pltest_j9sig_ext test to playlist.xml
-fixed error with _JVM_OPTIONS, all occurrences changed to JVM_OPTIONS

[ci skip]
Signed-off-by: Nikola Milijevic <nmilijev@uwaterloo.ca>

### CODE REVIEW

**Comment by nikolamilijevic1** [comment]
tested internally:
| BUILD | PLATFORM | TARGET  | BUILD_LIST | JAVA_VERSION |
--- | --- | --- |  --- |--- |
| Grinder/8107 | x86-64_linux | cmdLineTester_pltest_j9sig_ext | functional | 8 |
| Grinder/8108 | ppc64le_linux | cmdLineTester_pltest_j9sig_ext | functional | 8 |
| Grinder/8109 | s390x_linux | cmdLineTester_pltest_j9sig_ext | functional | 8 |
| Grinder/8111 | ppc64le_linux | cmdLineTester_pltest_j9sig_ext | functional | 11 |
| Grinder/8112 | s390x_linux | cmdLineTester_pltest_j9sig_ext | functional | 11 |
| Grinder/8113 | x86-64_linux | cmdLineTester_pltest_j9sig_ext | functional | 11 |

**Comment by nikolamilijevic1** [comment]
reviewers: @llxia @renfeiw

File: test/functional/cmdLineTests/pltest/playlist.xml [hunk 1]
```diff
@@ -367,4 +367,34 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) -DRESOURCES_DIR=$(Q)$(RESOURCES_DIR)$(Q) -DJVM_TEST_ROOT=$(Q)$(JVM_TEST_ROOT)$(Q) \
+	-DEXE=$(SQ)$(JAVA_SHARED_LIBRARIES_DIR)$(D)pltest -inlcude:j9sig_extended $(JVM_OPTIONS)$(SQ) -jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)pltest.xml$(Q) -verbose -explainExcludes \
+	-xids all,$(PLATFORM),$(VARIATION) -plats all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
+	$(TEST_STATUS)</command>
+		<!-- temporarily disable this test on windows; https://github.com/eclipse/openj9/issues/3212 -->
```
Comment (llxia) [inline test/functional/cmdLineTests/pltest/playlist.xml:381]: This comment is unrelated to `cmdLineTester_pltest_j9sig_ext `. Please remove it.

**Comment by llxia** [comment]
@nikolamilijevic1 I noticed that you changed the title to include WIP. Is it intended?
